src/api/libcellml/enums.h

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <memory>
+
+namespace libcellml {
+/**
+ * @brief The ItemType enum class.
+ *
+ * 
+ */
+// KRM not sure how to get this working with SWIG.
+enum class LIBCELLML_EXPORT ItemType
```

Comment:
I would suggest that you don't export an enum.

Reply:
Yeah, without this I get a different set of errors.  Hang on a mo ...